realtime_tools/include/realtime_tools/realtime_thread_safe_box.hpp

```diff
@@ -299,18 +299,11 @@ class RealtimeThreadSafeBoxBase
   mutable mutex_t lock_;
 };
 
-// Introduce some easier to use names
+// Provide specialisations for other mutex types
 
-// Provide specialisations for different mutex types
 template <typename T>
-using RealtimeThreadSafeBoxStandard = RealtimeThreadSafeBoxBase<T, std::mutex>;
-
-template <typename T>
-using RealtimeThreadSafeBoxRecursive = RealtimeThreadSafeBoxBase<T, std::recursive_mutex>;
-
-// This is the specialisation we recommend to use in the end
-template <typename T>
-using RealtimeThreadSafeBox = RealtimeThreadSafeBoxStandard<T>;
+using RealtimeThreadSafeBoxRecursive =
+  RealtimeThreadSafeBox<T, realtime_tools::prio_inherit_recursive_mutex>;
```

Comment:
May be like the DEFAULT_MUTEX 
You might need to define for the recursive mutex for the windows CI?

Reply:
ah you are right